### PR TITLE
Add page of external resources

### PR DESCRIPTION
**Pull request by jsignell**
Closes https://github.com/NASA-IMPACT/veda-analytics/issues/23
Looks like: 
image

### CODE REVIEW

File: external-resources.qmd [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "External Resources"
+subtitle: "List of resources for learning more about open source geospatial tools in Python"
```
Comment (j08lue) [inline external-resources.qmd:3]:
```suggestion
subtitle: "Learn more about tools and practice open source geospatial data science - a list of external resources to get you started"
```

File: external-resources.qmd [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+## Workflow resources
```
Comment (j08lue) [inline external-resources.qmd:6]:
```suggestion
## Master the basic workflow tools
```

File: external-resources.qmd [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* [Git - for managing code versions](https://docs.github.com/en/get-started/quickstart/github-flow)
+* [Conda - for managing dependencies](https://docs.conda.io/en/latest/)
+* [Jupyter - for running code](https://jupyter.org)
+
+## Geospatial courses/books
```
Comment (j08lue) [inline external-resources.qmd:11]:
```suggestion
## Take a course or read a book on geospatial data analysis
```

File: external-resources.qmd [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+* [Data Carpentry geospatial course](https://datacarpentry.org/geospatial-workshop/)
+* [UW course on Geospatial Data Analysis with Python](https://uwgda-jupyterbook.readthedocs.io/)
+* [Geographic data book - emphasis on vector data](https://geographicdata.science/book/intro.html)
+
+## Geospatial docs
```
Comment (j08lue) [inline external-resources.qmd:16]:
```suggestion
## Look through nicely curated lists of tools and resources
```

File: external-resources.qmd [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+* [Project Pythia - Pangeo's education hub](https://projectpythia.org/)
+* [Planetary Computer](https://planetarycomputer.microsoft.com/docs/overview/about)
+* [Geospatial Computing Platform library of training resources (by Python package)](https://crib.utwente.nl/manual/training-resources/README.html)
+
+## Materials from hack weeks
```
Comment (j08lue) [inline external-resources.qmd:21]:
```suggestion
## Find out what a Hack Week is and see how they teach best practice
```

File: external-resources.qmd [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+* [UW GeoHackWeek](https://github.com/geohackweek)
+* [UW OceanHackWeek](https://github.com/oceanhackweek/)
+
+## Blogs
```
Comment (j08lue) [inline external-resources.qmd:25]:
```suggestion
## Stay up-to-date through blogs
```

File: external-resources.qmd [hunk 7]
```diff
@@ -0,0 +1,26 @@
+---
+title: "External Resources"
+subtitle: "List of resources for learning more about open source geospatial tools in Python"
+---
+
```
Comment (j08lue) [inline external-resources.qmd:5]:
```suggestion

This list is intended for (aspiring) scientists who want to get started with cloud-based, collaborative, open geospatial data analysis, or those looking to refresh their knowledge. It is by no means complete, but contains pointers to more elaborate resources. We anticipate this list to evolve as our platform and use cases evolve. Suggestions for additional resources or topics are highly welcome.

```

**Review by j08lue — CHANGES_REQUESTED** [review]
I suggested a few changes, with the goal of making it very obvious to readers, who the list is intended for, and making the headings by themselves already instructional, so readers can see at a glance what type of resources or learnings we are recommending.

Please feel free to accept or adapt my suggestions, or let me know if you generally do not like the new format I am suggesting.

**Comment by jsignell** [comment]
No I think this is great! I was mostly thinking about what would be helpful groupings but lost sight of explaining how this page is actually intended to be used :facepalm: Thanks for the suggestions @j08lue!

**Review by j08lue — APPROVED** [review]

**Pull request merged**